agilent_infiniium_scope.py

```diff
@@ -409,67 +438,69 @@ def _parsePreamble(preamble):
                     ('maxBW',False),
                     ('minBW',False)]
     '''
-    preambleKeys = [('byteFormat',True),
-                ('dataType',True),
-                ('numPoints',True),
-                ('count',True),
-                ('xStep',True),
-                ('xFirst',True),
-                ('xRef',True),
-                ('yStep',True),
-                ('yOrigin',True),
-                ('yRef',True),
-                ('coupling',True),
-                ('xRange',True),
-                ('xLeftDisplay',True),
-                ('yRange',True),
-                ('yCenterDisplay',True),
-                ('date',True),
-                ('time',True),
-                ('model',True),
-                ('acquisitionMode',True),
-                ('percentTimeBucketsComplete',True),
-                ('xUnits',True),
-                ('yUnits',True),
-                ('maxBW',True),
-                ('minBW',True)]
-    preambleDict = {}
-    for key,val in zip(preambleKeys,preambleVals):
+    preamble_keys = [('byteFormat', True),
```

Comment:
This can be defined at the module level, rather than inside this function which rebuilds the list every time it is called. Also, I would prefer the following indentation:
```python
PREAMBLE_KEYS = [
    ('byteFormat', True),
    ...
]
```